### PR TITLE
Add stacking benchmark for CSI500 Alpha158

### PR DESCRIPTION
**Pull request by tanshaozong**
## Description

This PR will add a stacking model with LightGBM as base estimator for CSI500 index on Alpha158 dataset.

## How Has This Been Tested?

- [ ] Pass the test by running: `pytest qlib/tests/test_all_pipeline.py` under upper directory of `qlib`.
- [ ] If you are adding a new feature, test on your own test scripts.

## Screenshots of Test Results (if appropriate):
1. Pipeline test:
2. Your own tests:

## Types of changes

- [ ] Fix bugs
- [ ] Add new feature
- [ ] Update documentation

### CODE REVIEW

File: qlib/contrib/model/stacking.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from email.mime import base
```
Comment (you-n-g) [inline qlib/contrib/model/stacking.py:4]: What's the necessity to import it?

File: examples/benchmarks/Stacking/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Stacking
```
Comment (you-n-g) [inline examples/benchmarks/Stacking/README.md:1]: Please add more descriptions about it.
You can refer to the examples to others.

**Comment by you-n-g** [comment]
Please check the errors in CI.

File: qlib/contrib/model/stacking.py (hunk 1 above)
Comment (tanshaozong) [reply qlib/contrib/model/stacking.py]: Removed. This should be auto imported when typing base_estimator.

**Pull request closed without merging**